src/Swashbuckle.AspNetCore.SwaggerGen/XmlComments/XmlCommentsDocumentFilter.cs

```diff
@@ -12,21 +12,22 @@ public class XmlCommentsDocumentFilter : IDocumentFilter
         private const string MemberXPath = "/doc/members/member[@name='{0}']";
         private const string SummaryTag = "summary";
 
+        private readonly SwaggerGeneratorOptions _options;
         private readonly XPathNavigator _xmlNavigator;
 
-        public XmlCommentsDocumentFilter(XPathDocument xmlDoc)
+        public XmlCommentsDocumentFilter(SwaggerGeneratorOptions options, XPathDocument xmlDoc)
```

Comment:
As this type is public, this is a binary-breaking change that we couldn't merge until a major version (e.g. 7.0.0).

Is there a way you can make this compatible when `SwaggerGeneratorOptions` is optional so we can add a constructor overload instead?

```csharp
public XmlCommentsDocumentFilter(XPathDocument xmlDoc)
    : (xmlDoc, null)
{
}

public XmlCommentsDocumentFilter(XPathDocument xmlDoc, SwaggerGeneratorOptions options)
{
    // Set properties, allowing options to be null
}
```